clients/src/test/java/org/apache/kafka/common/utils/UtilsTest.java

```diff
@@ -158,6 +161,39 @@ public void testCloseAll() {
         }
     }
 
+    @Test
+    public void testReadFully() throws IOException {
+        RandomAccessFile raf = new RandomAccessFile(TestUtils.tempFile(), "rw");
```

Comment:
@ijuma Seems EasyMock cannot mock functions with side effects. Do you think it's okay I just use EasyMock to test throwing EOF and real files to test multiple reads?